src/fundus/publishers/us/rolling_stone.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class RollingStoneParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div.a-content > * > p.paragraph")
+        _summary_selector = CSSSelector("article > header > div.article-excerpt")
+        _subheadline_selector = CSSSelector("article > header > div.article-kicker")
```

Comment:
I could not find an example article for the subheadline selector, could you provide one in this comment?

Also, the subheadlines included in this article are not parsed properly:
https://www.rollingstone.com/politics/politics-news/donald-trump-indictments-lawsuits-legal-cases-explained-1234767770/

Reply:
For Fundus, `subheadlines` are headlines that separate paragraphs. In the example article i posted the subheadlines would be `Paying Off a Porn Star`, `Election Interference and Jan. 6`, ...
I would suggest `div.a-content h2.heading, div.a-content h2.c-gallery-vertical-featured-image__title`

Reply:
Thanks for clarifying. I added `div.a-content h2.heading`. For the other subheadlines, the class `h2.c-gallery-vertical-featured-image__title` could not be parsed because it is not in the original sourcecode (there, the content is only `h2` without any classes, maybe the class is added with a script before being shown in the browser). Instead, i selected via the identifier that is used by all of these lists with `div.a-content div#pmc-gallery-vertical h2`